components/PrimaryButton.js

```diff
@@ -10,7 +10,7 @@ export default class PrimaryButton extends React.Component {
 
   render() {
     return (
-         <View style={[styles.buttonContainer, {backgroundColor: this.props.backgroundColor}, {height: this.props.height}]}>
+         <View style={[styles.buttonContainer, {backgroundColor: this.props.backgroundColor}, {height: this.props.height}, {width: this.props.width}]}>
```

Comment:
If you're going to alter the `PrimaryButton`, while you're at it, please add shadow